model/profile/common.yaml

```
@@ -6,3 +6,19 @@ groups:
     attributes:
       - ref: profile.frame.type
         requirement_level: recommended
+
+  - id: profile.pprof
+    type: attribute_group
+    brief: >
+      Describes the relationship between pprof and a Profile.
```

Why not use the same brief description as in registry below, I think it's more specific and less vague:

```suggestion
      Attributes specific to pprof that help convert from pprof to Profiling signal.
```